packages/core-typings/src/IRoom.ts

```diff
@@ -94,7 +94,7 @@ export interface IRoom extends IRocketChatRecord {
 		items: [SidepanelItem, SidepanelItem?];
 	};
 
-	rolePrioritiesCreated?: boolean;
+	rolePrioritiesCreated?: number | boolean;
```

Comment:
I think we can mark boolean as deprecated since we will not add those anymore, also we should create a task for removing the boolean in the next major together with a migration.